.gitlab-ci.yml

```diff
@@ -286,25 +287,23 @@ build-linux-substrate:
           tee ./artifacts/substrate/VERSION;
       fi
     - sha256sum ./artifacts/substrate/substrate | tee ./artifacts/substrate/substrate.sha256
-    - printf '\n# building node-template\n\n'
-    - ./.maintain/node-template-release.sh ./artifacts/substrate/substrate-node-template.tar.gz
-    - cp -r .maintain/docker/substrate.Dockerfile ./artifacts/substrate/
     - sccache -s
 
+build-linux-node-template:
+  <<:                              *build-binary
+  script:
+      - ./.maintain/node-template-release.sh ./artifacts/substrate/substrate-node-template.tar.gz
+      - cp -r .maintain/docker/substrate.Dockerfile ./artifacts/substrate/
```

Comment:
Why do we do this here? And not in the build substrate job above?

Reply:
you're right, right now it's not effective to split these jobs

Reply:
actually, we are compiling it right there: https://gitlab.parity.io/parity/substrate/-/jobs/339822#L706 , during `cargo build`.
Can we use that artifact?

Reply:
I was speaking about the docker file. Putting the node template into its own job sounds reasonable.